{
    "type": "bundle",
    "id": "bundle--c237a740-f437-48a4-88f7-743af54b8f53",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--d8bb4224-65f4-4b01-b772-6926d2524f07",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2023-08-23T00:04:54.179614Z",
            "modified": "2023-08-23T00:04:54.179614Z",
            "name": "CVE-2022-25024",
            "description": "The json2xml package through 3.12.0 for Python allows an error in typecode decoding enabling a remote attack that can lead to an exception, causing a denial of service.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-25024"
                }
            ]
        }
    ]
}